### DISCUSSION TITLE
Classes not running

### DISCUSSION THREAD

**Discussion by Izaak3** [tailwindlabs/tailwindcss#13720] [Help]
I have been trying to change my class values, but the problem is that when I change the value to a different one from the one I first used, it doesn't run. Example: I've set a link's background color to pink-900, if I change it to pink-600 or any other value, it doesn't run!! Is there a way out?

**Comment by dario-baumberger** [comment]
Check the following:
- Has the css been recompiled?
- Are the colors included in the config (if the standard configuration has not been extended)?
- `content` has been set correctly in the configuration?

More information would be helpful.
